explorer/tests/test_exporters.py

```diff
@@ -75,4 +75,28 @@ def test_writing_excel(self):
 
         expected = b('PK')
 
-        self.assertEqual(res[:2], expected)
\ No newline at end of file
+        self.assertEqual(res[:2], expected)
+
+
+
+@unittest.skipIf(sys.version_info[0] > 2,  "not supported in this library version")
```

Comment:
I don't really like adding a utility function (in utils.py) because that would introduce a circular import. For the testing if pdf exporting we can just check if python version < 2 (the optional requirements should be installed anyway for testing). So I propose leaving it like this (I've just improved the message a bit).